src/control/system/attributes.go

```diff
@@ -0,0 +1,59 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package system
+
+import (
+	"github.com/pkg/errors"
+)
+
+type (
+	// SysAttrSetter defines an interface to be implemented by
+	// something that can set system properties.
+	SysAttrSetter interface {
+		SetSystemAttrs(props map[string]string) error
+	}
+	// SysAttrGetter defines an interface to be implemented by
+	// something that can get system properties.
```

Comment:
Probably best to maintain consistency between prop and attr terminology.

Reply:
Yeah, missed this in the rename.